test/manifests/manifests.test.ts

```diff
@@ -73,22 +73,23 @@ describe('manifest ingest', () => {
     })
   })
 
-  test('updateUnlManifests', async () => {
```

Comment:
Why were these tests changed so much?

Reply:
I would think the same keys would work in the testcase.

Reply:
The previous updateUNLManifests just get manifests from `xl.ripple.com`, while the new function get manifests from all of them, which would yield different results.

Reply:
`const unl: UNLBlob = await fetchValidatorList(await getFirstUNL('main'))`. This is what in the function before the change

Reply:
I fixed the test so that it could use the mock data and same keys